app/presenters/confirmation_email_presenter.rb

```diff
@@ -29,7 +29,7 @@ def confirmation_period
 
     view.distance_of_time_in_words(
       current_time,
-      current_time + Devise.confirm_within,
+      current_time + IdentityConfig.store.account_reset_wait_period_days.days.in_hours,
```

Comment:
I believe this presenter is used for confirming email addresses, and using `Devise.confirm_within` is correct.

Reply:
I will make sure to change it to `Devise.confirm_within`